src/plugins/ui_feature/collaborative.ts

```diff
@@ -32,8 +42,53 @@ export class CollaborativePlugin extends UIPlugin {
     );
   }
 
-  getClient() {
-    return this.session.getClient();
+  handle(cmd: Command) {
+    switch (cmd.type) {
+      case "JUMP_TO_CLIENT":
+        this.jumpToClient(cmd.clientId);
+        break;
+      case "SHOW_CLIENT_TAG":
+        this.showClientTag = true;
+        break;
+      case "HIDE_CLIENT_TAG":
+        this.showClientTag = false;
+        break;
+    }
+  }
+
+  jumpToClient(clientId: ClientId) {
+    const clients = this.getConnectedClients();
+    for (const client of clients) {
+      if (client.id === clientId && client.position) {
+        this.focusClient(client);
+        this.dispatch("ACTIVATE_SHEET", {
+          sheetIdTo: client.position.sheetId,
+          sheetIdFrom: this.getters.getActiveSheetId(),
+        });
+        this.dispatch("SCROLL_TO_CELL", { col: client.position.col, row: client.position.row });
+      }
+    }
+  }
+
+  focusClient(client: Client) {
+    if (client.focusTimeout) {
+      clearTimeout(client.focusTimeout);
+    }
+    client.focusTimeout = setTimeout(() => {
```

Comment:
I'm a bit skeptical about this timeout not being cleared properly when we no longer needs the model.
It's asking for trouble.
As an example, it means all models survive for at least 3 seconds in tests.

Could you move all this code in a local store (where you can register a cleanup callback with `this.onDispose`) ?